### ISSUE TITLE
Generate docs for Bicep modules

### ISSUE THREAD

**Issue by BernieWhite** [Azure/PSDocs.Azure#106]
We support docs for ARM Template or Bicep via ARM template. But should also support documentation for Bicep natively.

**Comment by jeffpatton1971** [comment]
Is there an ETA on when native bicep support will be available? If help is required I'm open just need some guidance on where to starts :)

**Comment by vegardhw** [comment]
> Is there an ETA on when native bicep support will be available? If help is required I'm open just need some guidance on where to starts :)

@BernieWhite Any news on this? 🙂

**Comment by SvenAelterman** [comment]
> It's not amazing but this mostly works:
> 
> bicep build file.bicep --outfile file.json
> Invoke-PSDocument -Module PSDocs.Azure -InputObject file.json -OutputPath out/docs/

Indeed, but it would be nice not to have to build yourself (and then delete the resulting JSON file). Even better would be if the parameters snippet would output bicepparam format instead of JSON.
